### ISSUE TITLE
Lunchtime Talk Autoresponse

### ISSUE THREAD

**Issue by pitchinvasion** [makersacademy/careers#2]

**Comment by pitchinvasion** [comment]
First task is to research a campaign management tool that can:

- Send out mass emails
- Reply with templated responses
- Re-send to those with no reply a different response

**Comment by pitchinvasion** [comment]
you can book me for lunchtime talks?

**Comment by spike01** [comment]
![notification_automation_wireframe](https://cloud.githubusercontent.com/assets/7307631/9759609/3043514e-56e6-11e5-93f7-36001b7462a0.jpg)

**Comment by nikeshashar** [comment]
This is where something like Mandrill or Customer.io would be awesome

**Comment by spike01** [comment]
Bullhorn actually does a lot of this, but I'll check out both of those too

**Comment by pitchinvasion** [comment]
Also this seems to handle segmentation based on responses https://www.campaignmonitor.com/features/subscribers/

**Comment by nikeshashar** [comment]
There's lots of solutions to the problem. We gotta find one and use the it across the business ideally to make it work properly.

**Comment by pitchinvasion** [comment]
Agreed. I'm catching up with will and sanj today maybe join at the beginning and we could decided on one to start testing?

**Comment by nikeshashar** [comment]
When is that meeting?

**Comment by pitchinvasion** [comment]
we just had it, we're going to delay until after we've trialled the recruitment software we're looking it

**Comment by tansaku** [comment]
@pitchinvasion @spike01 shall we move this back into ideas

**Comment by spike01** [comment]
:+1:

**Comment by rhiml11** [comment]
@isoworg @spike01 do we have a campaign management tool now? Are we working with Jordan and Arfah - this will be good for us when we get on to Webinar project

**Comment by nikeshashar** [comment]
@rhiml11 are you after a social media management tool or a google campaign tool ?
